VARIABLES.md

```diff
@@ -4,6 +4,14 @@ Below are the supported variables for the role confluent.variables
 
 ***
 
+### confluent_package_version
```

Comment:
I do not understand why this is added in this PR? Leftovers from something else?

Reply:
I want to "sneak this in" bc its missing. Do you feel this really needs a separate PR?